packages/io/amf-deserializer/package.json

```diff
@@ -39,6 +39,7 @@
   },
   "devDependencies": {
     "ava": "^0.25.0",
-    "nyc": "^10.3.2"
+    "nyc": "^10.3.2",
+    "@jscad/sample-files": "github:jscad/sample-files"
```

Comment:
I understand the logic but this has a serious side effect: it means that every time you install 
amf-serializer => IO => jscad you pull in the whole samples repo/package which is hugge

Reply:
I was trying to find a simple way to initiate tests for a single module. This would be very handy for making bug fixes to any of the modules.

Of course, the tests can be initiated from the specific directory. But this feels more ‘built-in’.

Not sure as to which is the best approach.

Reply:
I agree. I’m still scratching my head on this one.

The tests for DXF deserializer run just fine.

Do you see any differences?

Reply:
hmm I need to double check this, because you are right AND we should not have this dependency, will dig deeper in the next few days, have a bit more time available.

Reply:
The latest CI build works! I may have found a solution. :)
I'll continue to monitor as there are a few more changes to post.

Reply:
Seems to work just fine now. :)